packages/grid/x-grid-data-generator/src/renderer/renderEditProgress.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { GridCellParams } from '@material-ui/data-grid';
+import Slider from '@material-ui/core/Slider';
+import Tooltip from '@material-ui/core/Tooltip';
+import { createStyles, makeStyles } from '@material-ui/styles';
+import { debounce } from '@material-ui/core/utils';
+import { Theme } from '@material-ui/core/styles';
+import { createTheme } from '../../../_modules_/grid/utils/utils';
+
+const defaultTheme = createTheme();
+const useStyles = makeStyles(
+  (theme: Theme) =>
+    createStyles({
+      root: {
+        display: 'flex',
+        height: '100%',
+        width: '100%',
+        alignItems: 'center',
+        justifyContent: 'center',
+        padding: theme.spacing(1),
+      },
+    }),
+  { defaultTheme },
+);
+
+function ValueLabelComponent(props) {
+  const { children, open, value } = props;
+
+  return (
+    <Tooltip open={open} enterTouchDelay={0} placement="top" title={value}>
+      {children}
+    </Tooltip>
+  );
+}
+
+function EditProgress(props: GridCellParams) {
+  const classes = useStyles();
+  const { id, value, api, field } = props;
+  const [valueState, setValueState] = React.useState(Number(value));
+
+  const updateCellEditProps = React.useCallback(
+    (newValue) => {
+      const editProps = { value: newValue };
+      api.setEditCellProps({ id, field, props: editProps });
+    },
+    [api, field, id],
+  );
+
+  const debouncedUpdateCellEditProps = React.useMemo(() => debounce(updateCellEditProps, 60), [
+    updateCellEditProps,
+  ]);
+
+  const handleChange = (event, newValue) => {
+    setValueState(newValue);
+    debouncedUpdateCellEditProps(newValue);
+  };
+
+  React.useEffect(() => {
+    setValueState(Number(value));
+  }, [value]);
+
+  return (
+    <div className={classes.root}>
+      <Slider
```

Comment:
Maybe we can reuse the same style as the view? So we keep it consistent.

Reply:
Done.